### PR TITLE
test(VSwitch): Migrate to vue-test-utils

### PR DESCRIPTION
**Pull request by borisdayma**
## Description

Test was migrated to vue-test-utils:
* test file with vue-test-utils added in components/[component_name]/tests
* test file with avoriaz removed from test/unit/... folder

## Motivation and Context

Requirement for v2

## How Has This Been Tested?

unit

## Current issues:
* `.v-input--selection-controls__ripple` is not found while present when logging `wrapper.html()`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VSwitch/__tests__/VSwitch.spec.ts [hunk 1]
```diff
@@ -1,93 +1,112 @@
...
+    let ripple = wrapper.findAll('.v-input--selection-controls__ripple')
 
-    expect(ripple.length).toBe(0)
+    expect(ripple).toHaveLength(0)
 
     wrapper.setProps({ ripple: true })
 
-    ripple = wrapper.first('.v-input--selection-controls__ripple')
-
-    await wrapper.vm.$nextTick()
```
Comment (MajesticPotatoe) [inline packages/vuetify/src/components/VSwitch/__tests__/VSwitch.spec.ts]: if you add back in `await wrapper.vm.$nextTick()` on line 39, your tests will pass

Comment (borisdayma) [reply packages/vuetify/src/components/VSwitch/__tests__/VSwitch.spec.ts]: Great! Thanks!

**Comment by sh7dm** [comment]
@borisdayma is this PR WIP now?

**Comment by borisdayma** [comment]
@sh7dm Good point, I just edited the title

**Comment by sh7dm** [comment]
@borisdayma nice job!

**Pull request merged**